### PR TITLE
XAMARIN-44048 Need to resolve indention issues with UG documentation

### PR DESCRIPTION
**Pull request by HemalathaDuraisamy**
Hi Team,

I have resolved indention issues with UG documentation PdfViewer, Diagram,RichTextEditor and ProgressBar controls.Kindly review the changes.

Regards,
Hemalatha D.

### CODE REVIEW

File: Xamarin/Pdf-Viewer/Getting-Started.md [hunk 1]
```diff
@@ -187,11 +187,12 @@ namespace GettingStarted
 
 Add the following XAML code in the MainPage.xaml in the portable project.
 
-•	Includes the necessary namespace where the control class resides
-•	Set BindingContext of ContentPage to PdfViewerViewModel
-•	Includes PDF Viewer into the page
-•	Binds InputFileStream property of PDF Viewer control to PdfDocumentStream property of PdfViewerViewModel
+*	Includes the necessary namespace where the control class resides
```
Comment (ramkumarsync) [inline Xamarin/Pdf-Viewer/Getting-Started.md:190]: Remove tab here and use space instead.

File: Xamarin/Pdf-Viewer/Getting-Started.md [hunk 2]
```diff
@@ -187,11 +187,12 @@ namespace GettingStarted
...
+*	Set BindingContext of ContentPage to PdfViewerViewModel
+*	Includes PDF Viewer into the page
+*	Binds InputFileStream property of PDF Viewer control to PdfDocumentStream property of PdfViewerViewModel
 
+{% capture codesnippet1 %}
```
Comment (ramkumarsync) [inline Xamarin/Pdf-Viewer/Getting-Started.md:195]: Remove this change

File: Xamarin/Pdf-Viewer/Getting-Started.md [hunk 3]
```diff
@@ -584,7 +587,6 @@ N>This API is only applicable for UWP desktop. Changing the value of this API do
 ## How to get the list of annotations present in the PDF?
 
 By using `Annotations` property, You can get the list of annotations present in the PDF document.
-  .
 Refer the following code sample.
```
Comment (ramkumarsync) [inline Xamarin/Pdf-Viewer/Getting-Started.md:590]: Revert the changes

File: Xamarin/Pdf-Viewer/Touch-Interaction-Events.md [hunk 4]
```diff
@@ -33,6 +33,7 @@ The [`LongPressed`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewe
 * [`Position`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.TouchInteractionEventArgs.html#Syncfusion_SfPdfViewer_XForms_TouchInteractionEventArgs_Position) 			: The long pressed position with respect to the PDF Viewer's client area.
 * [`PagePosition`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.TouchInteractionEventArgs.html#Syncfusion_SfPdfViewer_XForms_TouchInteractionEventArgs_PagePosition) 			: The long pressed position with respect to the PDF page.
 
+{% capture codesnippet1 %}
```
Comment (ramkumarsync) [inline Xamarin/Pdf-Viewer/Touch-Interaction-Events.md:36]: Remove the whole file changes

**Review by Mohanaraja — APPROVED** [review]
code changes are fine

**Review by NavaneethS — APPROVED** [review]

**Comment by NavaneethS** [comment]
rebuild
